src/coreclr/src/tools/Common/JitInterface/CorInfoBase.cs

```diff
@@ -572,7 +572,7 @@ static CorInfoIntrinsics _getIntrinsicID(IntPtr thisHandle, IntPtr* ppException,
             var _this = GetThis(thisHandle);
             try
             {
-                return _this.getIntrinsicID(method, pMustExpand);
+                return _this.GetIntrinsicId(method, pMustExpand);
```

Comment:
This is auto-generated file, do not exit it directly. It is intentional that the casing matched the casing of the C++ method.